app/controllers/simplified-account/settings/email-notifications/email-notifications.controller.js

```diff
@@ -1,7 +1,17 @@
 const { response } = require('../../../../utils/response')
+const humaniseEmailMode = require('../../../../utils/humanise-email-mode')
 
 function get (req, res) {
-  return response(req, res, 'simplified-account/settings/email-notifications/index')
+  const service = req.service
+  const account = req.account
+
+  const context = {
+    emailCollectionMode: humaniseEmailMode(account.email_collection_mode),
+    confirmationEmailEnabled: account.email_notifications.PAYMENT_CONFIRMED.enabled,
```

Comment:
is there a reason why we check for the existence of `email_notifications` for `REFUND_ISSUED` but not `PAYMENT_CONFIRMED`?

Reply:
I've not thought about it. Initially I just ported over the [existing functionality](https://github.com/alphagov/pay-selfservice/blob/master/app/controllers/email-notifications/email-notifications.controller.js#L89-L98).